src/process.cr

```diff
@@ -44,6 +44,15 @@ class Process
     nil
   end
 
+  # Sends a *signal* to the processes identified by the given *pids*.
+  def self.signal(signal : Signal, *pids : Int)
```

Comment:
Since we're changing the name anyway: I really don't like that this method takes multiple pids. This behaviour isn't even tested. Should this method just be `def self.signal(signal : Signal, pid : Int) : Nil`?

Reply:
Absolutely. There seems to be no reason why receiving multiple pids would be better than iterating externally.